app/javascript/components/provider-form/index.jsx

```diff
@@ -176,7 +177,7 @@ const ProviderForm = ({
       // Construct the full form data with all the necessary items
       const data = {
         ...rest,
-        endpoints: trimFieldValue(endpoints),
+        endpoints,
```

Comment:
The method `trimFieldValue` should receive an object, modify it, and return the modified object.

But, while submitting, the method received an array from `endpoints`.

I don't remember why I added the method only for `endpoints`.

I guess the below change would work. Could you please check...

```
const data = trimFieldValue({
        ...rest,
        endpoints,
        authentications,
        ...(edit ? undefined : { type }),
        ddf: true,
      });
```